mmv1/products/accesscontextmanager/api.yaml

```diff
@@ -899,6 +905,220 @@ objects:
                 The list of APIs usable within the Service Perimeter.
                 Must be empty unless `enableRestriction` is True.
               item_type: Api::Type::String
+          - !ruby/object:Api::Type::Array
+            name: 'ingressPolicies'
+            description: /
+              List of `IngressPolicies` to apply to the perimeter. A perimeter may
+              have multiple `IngressPolicies`, each of which is evaluated
+              separately. Access is granted if any `Ingress Policy` grants it.
+              Must be empty for a perimeter bridge.
+            at_least_one_of:
+              - status.0.resources
+              - status.0.access_levels
+              - status.0.restricted_services
+              - status.0.ingress_policies
+              - status.0.egress_policies
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+              - !ruby/object:Api::Type::NestedObject                  
+                name: 'ingressFrom'
+                description: /
+                  Defines the conditions on the source of a request causing this `IngressPolicy`
+                  to apply.
+                at_least_one_of:
+                  - status.0.ingress_policies.ingressFrom
```

Comment:
Unfortunately I don't believe this will work. Because ingress_policies is an array we would have to hard-code the index here, but since it is an array that allows multiple entries we could only ever check the first entry with a hard coded index.